lib/src/combase.dart

```diff
@@ -15,13 +15,19 @@ import 'macros.dart';
 import 'utils.dart';
 import 'win32/ole32.g.dart';
 
+/// Alias for a Pointer to COM vtable.
+typedef LPVTBL = Pointer<Pointer<IntPtr>>;
```

Comment:
I wonder if we should name this typedef `VTablePointer` (i.e. Dart-style casing)? Naming it in uppercase makes it look like this is a Win32 type like `LPWSTR` or `HANDLE`, when as far as I know it doesn't exist in windows.h.

Reply:
Makes sense, done.